### PR TITLE
arm64: dts: rk3588-fxblox-rk1: add edp and user leds

### PR DESCRIPTION
**Pull request by mahdichi**
arm64: dts: rk3588-fxblox-rk1: add edp and user leds

### CODE REVIEW

File: arch/arm64/boot/dts/rockchip/rk3588-fxblox-rk1.dts [hunk 1]
```diff
@@ -597,8 +641,8 @@
 
 &display_subsystem {
 	clocks = <&hdptxphy_hdmi_clk0>, <&hdptxphy_hdmi_clk1>;
-	clock-names = "hdmi0_phy_pll", "hdmi1_phy_pll";
-
+	clock-names = "hdmi0_phy_pll", "hdmi1_phy_pll";	
+
```
Comment (amazingfate) [inline arch/arm64/boot/dts/rockchip/rk3588-fxblox-rk1.dts]: These two lines are adding unintended changes?

**Review by amazingfate — CHANGES_REQUESTED** [review]

File: arch/arm64/boot/dts/rockchip/rk3588-fxblox-rk1.dts (hunk 1 above)
Comment (mahdichi) [reply arch/arm64/boot/dts/rockchip/rk3588-fxblox-rk1.dts]: no its ok

Comment (amazingfate) [reply arch/arm64/boot/dts/rockchip/rk3588-fxblox-rk1.dts]: What's the reason of adding extra spaces to these two line?

Comment (mahdichi) [reply arch/arm64/boot/dts/rockchip/rk3588-fxblox-rk1.dts]: I fixed it.

**Review by amazingfate — APPROVED** [review]

**Pull request merged**